src/commands/Information/userinfo.ts

```diff
@@ -132,7 +133,7 @@ export default class extends Command {
 			const msg = await context.channel.send({ embeds: [embed], components });
 
 			const filter = (i: any) => !i.bot;
-			const collector = msg.createMessageComponentCollector({ filter, time: 30000 });
+			const collector = msg.createMessageComponentCollector({ filter, time: 1000 });
```

Comment:
This makes it impossible to pick whatever action the user wants to perform. Change back to 30 seconds